src/app/js/i18n/translations/fr.js

```diff
@@ -167,6 +167,11 @@ export default {
     show_publication_errors: 'Erreurs',
     model: 'Modèle',
     overview: 'Syndication',
+    overviewNone: 'Aucun',
+    overviewTitle: 'RESSOURCE - Titre',
+    overviewDescription: 'RESSOURCE - Description',
+    overviewDatasetTitle: 'COLLECTION - Titre',
```

Comment:
What?!? But there is also a `cover_collection` key which translation is `Différent pour chaque ressource`. Collection is a  cover level, doesn't it?

In fact, `error_cover_invalid_cover` says `Cover must be either dataset or collection`. That means that Collection is different from Dataset.

One can't translate `Dataset` with `Collection`.

Reply:
perhaps, the best way, is to do not translate DATASET & RESOURCE prefix in french...